getting-started/README.adoc

```diff
@@ -160,16 +161,72 @@ NOTE: SASL/PLAIN authentication is also available for tools and libraries that d
 
 +
 You'll use the service account information that you saved to configure your application to connect to your Kafka instances when you're ready. For example, if you plan to use https://github.com/edenhill/kafkacat[Kafkacat^] to interact with your Kafka instance, you'll use this information to set your bootstrap server and client environment variables.
-. To review your service account information, reset your credentials, or delete the service account, go to the *Streams for Apache Kafka* > *Service Accounts* page in the web console.
+. To review your service account information, reset your credentials, or delete the service account, use the left navigation menu to go to the *Service Accounts* page.
 
 .Verification
 ifdef::qs[]
 * Did you save the bootstrap server, client credentials, and authentication token endpoint to a secure location?
-* Did you verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page?
+* Did you verify that your service account was successfully created in the *Service Accounts* page?
 endif::[]
 ifndef::qs[]
 . Verify that the bootstrap server, client credentials, and authentication token endpoint are saved to a secure location.
-. Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
+. Verify that your service account was successfully created in the *Service Accounts* page.
+endif::[]
+
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in the Access Control List (ACL) of the Kafka instance. {product} uses ACLs provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* You've created a service account that you want to allow to access the running Kafka instance.
+
+.Procedure
+. In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
+. Click the *Access* tab to view the current ACL for this instance.
+. Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
+. Under *Assign Permissions*, use the drop-down menus to set the following permissions for the specified resource types for this service account. Click *Add* to add each new resource permission.
```

Comment:
Per an earlier comment, I suggest "use the drop-down menus to set the permissions **shown in the table**..."